pkgs/applications/graphics/vpv/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, libpng
+, libtiff
+, libjpeg
+, SDL2
+, gdal
+, octave
+, rustPlatform
+, cargo
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vpv";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "kidanger";
+    repo = "vpv";
+    rev = "v${finalAttrs.version}";
+    sha256 = "0cphgq1pqmwrjdmq524j5y522iaq6yhp2dpjdv0a3f9558dayxix";
+  };
+
+  cargoRoot = "src/fuzzy-finder";
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    src = finalAttrs.src;
+    sourceRoot = "source/src/fuzzy-finder";
+    hash = "sha256-CDKlmwA2Wj78xPaSiYPmIJ7xmiE5Co+oGGejZU3v1zI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    rustPlatform.cargoSetupHook
+    cargo
+  ];
+
+  buildInputs = [
+    libpng
+    libtiff
+    libjpeg
+    SDL2
+    gdal
+    octave
+  ];
+
+  cmakeFlags = [
+    "-DUSE_GDAL=ON"
+    "-DUSE_OCTAVE=ON"
+    "-DVPV_VERSION=v${finalAttrs.version}"
+    "-DBUILD_TESTING=ON"
+  ];
+
+  doCheck = true;
```

Comment:
fails to build

```
---- tests::test_fuzzy_path_finder_multiple_results_spaces stdout ----
thread 'tests::test_fuzzy_path_finder_multiple_results_spaces' panicked at 'assertion failed: `(left == right)`
  left: `["fo ob arimage.png", "foobarm age.p ng", "f oobarimae.jpg"]`,
 right: `["fo ob arimage.png", "foobarm age.p ng"]`', src/lib.rs:300:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Indeed the tests sometimes fail, the issue is known but I forgot about it. I've removed the `doCheck`, and I will put it back once this is fixed upstream.